pkg/config/config.go

```diff
@@ -74,11 +74,11 @@ type ComponentSettings struct {
 
 	Envs EnvVarList `yaml:"Envs,omitempty"`
 
-	Url *[]ConfigUrl `yaml:"Url,omitempty"`
+	URL *[]ConfigURL `yaml:"Url,omitempty"`
```

Comment:
@kadel do we wanna change the `yaml:Url` to `yaml:URL`. This will break the backward compatibility of the config file.

Reply:
lets keep it as it is